jobs/migrations/0001_initial.py

```diff
@@ -17,14 +17,17 @@ class Migration(migrations.Migration):
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('title', models.CharField(max_length=100)),
                 ('description', models.CharField(max_length=255)),
-                ('location', models.CharField(max_length=100)),
+                ('location', models.CharField(blank=True,
```

Comment:
what does `blank` means? (in the sense of location)

Reply:
If the location is not one of the 3 options given then when creating the job it can be left empty. If it is irrelevant I will remove it. What do you think?

Reply:
@yuvalk ?

Reply:
you need to decide, I just pointed out, that maybe it make sense not to allow blanks
or maybe give it an actual name '4', 'remotee' or IDK.
or '4', 'other' or '4', 'unspecified'
you also need to think and decide how will these appear in searches
if I'm searching for 'Haifa' will I also see all the unspecified ones?

Reply:
Ok I will add the option: '4' = 'Unspecified' . The search function will only return the jobs with specified locations without the Unspecified ones (Tel Aviv, Jerusalem, Haifa). Option Tel Aviv will only return Tel Aviv jobs. Option Jerusalem will only return Jerusalem  jobs. Option Haifa will only return Haifa jobs. Option Unspecified will only return Unspecified jobs. Last option will be to display all Jobs regardless of location.

Reply:
@yuvalk Fixed